src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -9,6 +9,8 @@ export type Project = {
   id: string;
   slug: string;
   isMember: boolean;
+
+  isBookmarked?: boolean;
```

Comment:
Might not need to be optional? AFAIK we always get an `isBookmarked` value for a project

Reply:
done.